variantEffectPredictor.wdl

```diff
@@ -0,0 +1,253 @@
+version 1.0 
+workflow variantEffectPredictor {
+  input {
+    File vcfFile
+    File vcfIndex
+    File? targetBed
+    File? custom
+    Boolean? toMAF
+    String tumorName
+    String normalName
+  }
+
+  String customCommand = if defined(custom) == true then "true" else "false"
+
+  if (defined(targetBed) == true) {
```

Comment:
Can the task calls be collapsed into:
1) if defined(targetBed) call targetBedTask
2) call vep {      input: 
vcfFile = select_first([targetBedTask.vcfFile,vcfFile],
vcfIndex = select_first([targetBedTask.vcfIndex, vcfIndex]),
...
3) if toMAF call vcf2maf

And then outputs modified to:
 File outputVcf = vep.vepVcfOutput
 File? outputMaf = vcf2maf.mafOutput